src/wdio/services/TerraService.js

```diff
@@ -64,4 +64,21 @@ export default class TerraService {
     /* Set the viewport size before the spec begins.  */
     viewportHelpers.setViewport(global.browser.options.formFactor);
   }
+
+  // To more passively support code splitting in terra dev site, wait for data to load before progressing with the test.
+  // eslint-disable-next-line class-methods-use-this
+  afterCommand(commandName) {
+    if (commandName === 'refresh' || (commandName === 'url')) {
+      if (global.browser.isExisting('[data-terra-dev-site-loading]')) {
+        try {
+          global.browser.waitUntil(() => (
```

Comment:
I have a feeling this is could  break in IE since we are passing a function. We double check this.

Reply:
This isn't a browser.execute function, It's borrowed from here:
https://github.com/webdriverio-boneyard/v4/blob/master/lib/commands/waitForExist.js
but we don't need all the reversing logic.

Reply:
Ah so we didn't want to do:
 browser.waitForExist('[data-terra-dev-site-content]', 5000);